lib/screens/explore_screens/flag/interactor/viewmodels/flag_bloc.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+
+import 'package:async/async.dart';
+import 'package:bloc/bloc.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/mappers/flagged_users_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/usecases/get_flagged_users_use_case.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_event.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_state.dart';
```

Comment:
Please use `part` and `part of` directives instead package import for bloc files.
For more detailed explanatio of this choice:
**https:// github.com/ felangel/ bloc/ issues/ 1554**
Also all of our blocs follow that style.

Reply:
done